### PR TITLE
readme improvements

### PR DESCRIPTION
**Pull request by 72squared**
Some simple improvements to the readme. I am sure we will continue making improvements in this area over time but I'm taking a quick swag at it now until we can do a more complete job later on.

This is related to issue #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,108 @@
...
+libraryDependencies += "com.github.scoquelin" %% "arugula-api" % arugulaVersion
+libraryDependencies += "com.github.scoquelin" %% "arugula-core" % arugulaVersion
+```
+
+This will add the Arugula API and core packages to your project along with the Lettuce dependency.
+If you want to only use the API in your package, you can exclude the core package from the dependencies
+and code only against the API. This decouples the API from the core implementation and allows you to write
+your own implementation of the commands or write an adapter for another underlying Redis client.
+
+In the future we could add support for a Jedis wrapper or other JVM-based Redis clients. That said,
```
Comment (scoquelin) [inline README.md]: Not sure about that part, Jedis does not support asynchronous calls and they dropped the ball for supporting it : https://github.com/redis/jedis/pull/713

Comment (72squared) [reply README.md]: k. will remove that comment.

Comment (72squared) [reply README.md]: I could just say:

> In the future we could add support for other JVM-based Redis clients.

Comment (72squared) [reply README.md]: got rid of that part and re-worded it to just focus on using Lettuce documentation to provide more insight on how to use Arugula.

**Review by scoquelin — APPROVED** [review]

**Pull request merged**